mcstatus/bedrock_status.py

```diff
@@ -4,11 +4,11 @@
 import socket
 import struct
 from time import perf_counter
-from typing import Optional
 
 import asyncio_dgram
 
 from mcstatus.address import Address
+from mcstatus.status_response import BedrockStatusResponse as BedrockStatusResponse  # noqa: I250
```

Comment:
This would be the same, for type checker, as just add `BedrockStatusResponse` to `__all__`. But because there is no `__all__` here, easier would be just add `some as some`.

I actually mention this somewhere...

Reply:
I'd way rather see an `__all__` added than using this weird `import x as x` practice.

Reply:
I could add an comment near this import. Just `__all__` is explicit there.

Reply:
I disagree, we should use `__all__`, not weird workarounds like this, even if well commented.

Reply:
Fixed in 598d2d5764db2aa0466a6dcbbf17915b006ae019.